CHANGELOG.md

```diff
@@ -153,7 +154,7 @@ showing the source changes from the previous version.
     - Error on warning from devtools::check [#1253](https://github.com/opendp/opendp/pull/1253)
 
 
-## [0.9.2-dev](https://github.com/opendp/opendp/compare/v0.9.1...HEAD) - TBD
+## [0.9.2](https://github.com/opendp/opendp/compare/v0.9.1...v0.9.2) - 2024-02-08
```

Comment:
I don't this this tag was made: the link doesn't really work: https://github.com/opendp/opendp/compare/v0.9.1...v0.9.2

- See #1628